packages/react/src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -405,14 +405,16 @@ class OverflowMenu extends Component {
         (event) => {
           const { target } = event;
           const { current: triggerEl } = this._triggerRef;
-          if (
-            !menuBody.contains(target) &&
-            triggerEl &&
-            !target.matches(
-              `.${prefix}--overflow-menu,.${prefix}--overflow-menu-options`
-            )
-          ) {
-            this.closeMenu();
+          if (target.matches) {
```

Comment:
```suggestion
          if (typeof target.matches === 'function') {
```

Which is donehttps://github.com/carbon-design-system/carbon/blob/7a95a7d46c2a36c19fffafcbf1153c7ba19f7f7f/packages/components/src/globals/js/misc/event-matches.js#L19 (function type check).